Dockerfile

```diff
@@ -257,6 +257,7 @@ COPY --from=gcr.io/projectsigstore/cosign:v1.3.1@sha256:3cd9b3a866579dc2e0cf2fde
 COPY ./Dockerfile.d/test-integration-etc_containerd-stargz-grpc_config.toml /etc/containerd-stargz-grpc/config.toml
 COPY ./Dockerfile.d/test-integration-ipfs-offline.service /usr/local/lib/systemd/system/
 COPY ./Dockerfile.d/test-integration-buildkit-nerdctl-test.service /usr/local/lib/systemd/system/
+RUN mkdir /temp-init && wget https://github.com/krallin/tini/releases/download/v0.19.0/tini-amd64 && cp tini-amd64 /temp-init/nerdctl-init && chmod +x /temp-init/nerdctl-init && rm tini-amd64
```

Comment:
Please use curl, and validate the digest

Reply:
Got it